### DISCUSSION TITLE
help!

### DISCUSSION THREAD

**Discussion by LightWardenX** [FabricMC/fabric-api#3553] [Player support]
### Description

Every time I install a terrain generation mod and I try to start up a world or create a new one, it never starts. its either stuck at zero or 100%, or the application stops responding as soon as I press start game.

**Comment by Linguardium** [comment]
That section labeled "Minecraft Logs" is where you were supposed to post a link to your latest.log and if you have one your crashlog after uploading them to a paste site like https://mclo.gs/
